index.html

```diff
@@ -5,11 +5,85 @@
+                    <a class="white-color no-text-decoration " href="#">Support</a>
+                    <form class="parent-element">
+                        <input type="text" name="research-bar"><i class="fas fa-search white-color child-element"></i>
+                    </form>
+                </nav>
+            </div>
+            <div class="main-heading">
+                <h1 class="white-color">What will your verse be?</h1>
+                <h2 class="white-color">Everyone has something to share. See how people are using<br>
+                    iPad in their unique ways to add to the world's story.</h2>
+                <h3 class="parent-element white-color"><a class="no-text-decoration white-color" href="#">watch the
+                        "Your Verse" film and explore the story<i
+                            class="far fa-play-circle fa-lg child-element"></i></a></h3>
+            </div>
+        </header>
+        <main>
+            <div class="main-content">
+                <div class="main-content-box1 image-width">
+                    <a href="#"><img class="image-width" src="images/forward-thinking.jpg" alt="image of iPhone 5S"></a>
+                </div>
+                <div class="main-content-box2 image-width">
+                    <a href="#"><img class="image-width" src="images/for-the-color-full.jpg"
+                            alt="image of iPhone 5C"></a>
+                </div>
+                <div class="main-content-box3 image-width">
+                    <a href="#"><img class="image-width" src="images/HBD.jpg" alt="image of apple birthday"></a>
+                </div>
+                <div class="main-content-box4 image-width">
+                    <a href="#"><img class="image-width" src="images/supplier.jpg" alt="image of Apple Supplier"></a>
+                </div>
+            </div>
+        </main>
+        <footer>
+            <div class="footer-link">
+                <div class="footer-link-right parent-element">
+                    <a class="no-text-decoration footer-text-size link-color " href="#">Site Map</a>
+                    <a class="no-text-decoration footer-text-size link-color " href="#">Hot News</a>
+                    <a class="no-text-decoration footer-text-size link-color " href="#">RSS Feeds </a>
+                    <a class="no-text-decoration footer-text-size link-color " href="#">Media Info</a>
+                    <a class="no-text-decoration footer-text-size link-color " href="#">Environment</a>
+                    <a class="no-text-decoration footer-text-size link-color " href="#">Job Opportunities</a>
+                    <a class="no-text-decoration footer-text-size link-color " href="#">Contact Us</a>
+                    <button class="child-element" type="button"><img class="us-flag" src="images/icon-usa.png"
+                            alt="usa-flag-icon"></button>
+                </div>
+                <div class="footer-link-left">
```

Comment:
- [optional ] The whole footer could have more padding.

| Yours | Original |
|:---:|:---:|
| ![image](https://user-images.githubusercontent.com/61479648/92503542-c5fa4f80-f1d7-11ea-98dc-310ed557ee58.png)|![image](https://user-images.githubusercontent.com/61479648/92503578-d7435c00-f1d7-11ea-85b8-387c2ba5c2e4.png)  |